C_tasks-here/bentii/C_codes-task4.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+int main(){
+int sumOfNumber, firstNumber, secondNumber
+;
+	printf("Enter the first number:");
+
+	scanf("%d", &firstNumber);
+
+	printf("Enter the second number:");
+
+	scanf("%d", &secondNumber);
+
+	sumOfNumber = firstNumber + secondNumber;
+
+	printf("The total of the numbers is: %d",sumOfNumber);
+
+	return 0
```

Comment:
you forgot to add semi colon after 0. i.e return 0;
Kindly fix

Reply:
okay...